internal/dkg/store.go

```diff
@@ -24,42 +24,107 @@ type BoltStore struct {
 }
 
 const BoltFileName = "dkg.db"
+const dkgFileName = "dkg.toml"
+const dkgStagedFileName = "dkg.staged.toml"
 const BoltStoreOpenPerm = 0660
 const DirPerm = 0755
 
 var stagedStateBucket = []byte("dkg")
 var finishedStateBucket = []byte("dkg_finished")
 
-func NewDKGStore(baseFolder string, options *bolt.Options) (*BoltStore, error) {
-	err := os.MkdirAll(baseFolder, DirPerm)
+type FileStore struct {
+	baseFolder string
+	log        log.Logger
+}
+
+func NewDKGStore(baseFolder string) (*FileStore, error) {
+	err := os.MkdirAll(baseFolder, 0770)
+	if err != nil {
+		return nil, err
+	}
+	return &FileStore{
+		baseFolder: baseFolder,
+		log:        log.New(nil, log.DebugLevel, true),
```

Comment:
this log level ought to be set by the caller - users can run their node with `-v` to enable debug logging

Reply:
Oh, honestly this is just copypasta which I didn't pay attention to. I'll sort this out